addons/csi/openstack/controllerplugin.yaml

```diff
@@ -31,6 +25,9 @@
 {{ if eq .Cluster.MajorMinorVersion "1.21" }}
 {{ $version = "v1.20.3 "}}
 {{ end }}
+{{ if eq .Cluster.MajorMinorVersion "1.22" }}
+{{ $version = "v1.21.1 "}}
```

Comment:
Why are we one minor version behind? I think we should use 1.22.0 for 1.22 clusters (https://github.com/kubernetes/cloud-provider-openstack/releases/tag/v1.22.0).